tests/test-miss-handlers.lua

```diff
@@ -40,4 +40,8 @@ local w = wibox()
 w.foo = "bar"
 assert(w.foo == "bar")
 
+-- Test if read-only properties really are read-only
+screen[1].clients = 42
+assert(screen[1].clients ~= 42)
```

Comment:
Should there be a warning or even error in this case?
Something like `cannot set read-only attribute: clients`?